msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityParameters.java

```diff
@@ -27,7 +29,18 @@ public Set<String> scopes() {
 
     @Override
     public ClaimsRequest claims() {
-        return null;
+        if (claims == null || claims.isEmpty()) {
```

Comment:
suggestion: If the null/empty check was move to the builder, then the rest of the library could just ignore it and wouldn't have to throw an exception

Reply:
Yeah noticed it when the tests started failing. Always takes a bit to switch to Java style from .Net